### PR TITLE
SPCTR-2752: Shift useDeviceType function in higher order component

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Shifting useDeviceType to higher order component.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,4 +90,4 @@
     "package": "rm *.zip && wp dist-archive . ultimate-addons-for-gutenberg",
     "packages-update": "wp-scripts packages-update"
   }
-}
\ No newline at end of file
+}
```
Comment (rahulvarma722) [inline package.json:93]: Remove unwanted changes

File: src/blocks/faq/render.js [hunk 2]
```diff
@@ -23,8 +22,7 @@ const Render = ( props ) => {
 	}, [] );
 
 	props = props.parentProps;
-	const deviceType = useDeviceType();
-	const { attributes } = props;
+	const { attributes, deviceType, } = props;
```
Comment (rahulvarma722) [inline src/blocks/faq/render.js:25]: Remove comma after devicetype

Comment (rajmohanreddy774) [reply src/blocks/faq/render.js]: Removed

File: package.json (hunk 1 above)
Comment (rajmohanreddy774) [reply package.json]: Removed

**Review by rahulvarma722 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**